### ISSUE TITLE
Login page needs to be updated

### ISSUE THREAD

**Issue by bykidi** [DirtDiglett/Pressure-for-Steam#469]
Vanilla http://imgur.com/a/JAJ0E
Pressure http://imgur.com/a/h0tTl

**Comment by bykidi** [comment]
hello? :D

**Comment by Js41637** [comment]
hi

**Comment by FlamingFox911** [comment]
Hello, I looked at this a while back and didn't want to say anything, but...
Basically, this requires a fair amount of work since this uses the .res file instead of .layout. For one, the gui helper (F6 in -dev mode) doesn't correctly select the region that is expanding, meaning I don't know exactly which function is causing the expansion, or how to prevent it.
I'm looking into it now, but it will take a while.

**Comment by Js41637** [comment]
Yeah i don't know how to fix this, gonna have to wait for DirtDiglett to fix it.

**Comment by DirtDiglett** [comment]
> gonna have to wait for DirtDiglett to fix it.

HAHAHAHHAHAHAHAHAHAHHAHAHAHAHHAHAHAHAHAHAHHAHAHAHAHHAHAHAHAHAHAHHAHAHAHAHHAHAHAHAHAHAHHAHAHAHAHHAHAHAHAHAHAHHAHAHAHAHHAHAHAHAHAHAHHA

**Comment by Js41637** [comment]
hurry up

**Comment by avsaponchik** [comment]
ye, hurry up :D

**Comment by Js41637** [comment]
I will look into it again
